images/README.md

```diff
@@ -78,9 +78,14 @@ future changes by simply running `terraform apply
 | provisionec2amicreateroles_role_name | The name to assign the IAM role (as well as the corresponding policy) with the ability to create IAM roles that can create AMIs in the Images account. | string | `ProvisionEC2AMICreateRoles` | no |
 | provisionkmskeys_role_description | The description to associate with the IAM role (as well as the corresponding policy) that allows sufficient permissions to provision KMS keys in the Images account. | string | `Allows sufficient permissions to provision KMS keys in the Images account.` | no |
 | provisionkmskeys_role_name | The name to assign the IAM role (as well as the corresponding policy) that allows sufficient permissions to provision KMS keys in the Images account. | string | `ProvisionKMSKeys` | no |
+| provisionthirdpartybucketreadroles_role_description | The description to associate with the IAM role (as well as the corresponding policy) with the ability to create IAM roles that can read objects in the third-party file storage S3 bucket in the Images account. | string | `Allows creation of IAM roles that can read objects in the third-party file storage S3 bucket in the Images account.` | no |
+| provisionthirdpartybucketreadroles_role_name | The name to assign the IAM role (as well as the corresponding policy) with the ability to create IAM roles that can read objects in the third-party file storage S3 bucket in the Images account. | string | `ProvisionThirdPartyBucketReadRoles` | no |
 | provisionvpcs_policy_description | The description to associate with the IAM policy that allows sufficient permissions to provision VPCs (and related resources) in the Images account. | string | `Allows sufficient permissions to provision VPCs (and related resources) in the Images account` | no |
 | provisionvpcs_policy_name | The name to assign the IAM policy that allows sufficient permissions to provision VPCs (and related resources) in the Images account. | string | `ProvisionVPCs` | no |
 | tags | Tags to apply to all AWS resources created. | map(string) | `{}` | no |
+| third_party_bucket_name_prefix | The prefix to use to name the S3 bucket for storing third-party files.  The bucket will be named with this prefix plus the account type (e.g. production or staging). | string | `cisa-cool-third-party` | no |
+| thirdpartybucketread_role_description | The description to associate with the IAM role (as well as the corresponding policy) that allows sufficient permissions to read objects in the third-party file storage S3 bucket in the Images account. | string | `Allows sufficient permissions to read objects in the third-party file storage S3 bucket in the Images account.` | no |
```

Comment:
I don't think that `thirdpartybucketread_role_description` and `thirdpartybucketread_role_name` actually exist.

Reply:
Whoops- good catch.  I was rushing and in my haste, I copied/pasted incorrectly.  See https://github.com/cisagov/cool-accounts/pull/47/commits/bfcae42ae29ff1b1439983f35421c60f5db71446.